man/meson.build

```
@@ -1,4 +1,3 @@
 install_man('trietool.1')
 
-mandir_abs = get_option('prefix') / get_option('mandir')
-meson.add_install_script(sh, '-c', 'cd @0@ && ln -s trietool.1 trietool-0.2.1 || true'.format(mandir_abs / 'man1'))
+meson.add_install_script(sh, '-c', 'cd ${MESON_INSTALL_DESTDIR_PREFIX}/@0@/man1 && ln -s trietool.1 trietool-0.2.1 || true'.format(get_option('mandir')))
```

Since the minimum version was bumped from 0.53, and is now >=0.61.0, you can use:

```suggestion
install_symlink('trietool-0.2.1', pointing_to: 'trietool.1', install_dir: get_option('mandir') / 'man1')
```